sys/include/crypto/ciphers.h

```diff
@@ -126,6 +126,9 @@ typedef struct {
  * @param cipher_id  cipher algorithm id
  * @param key        encryption key to use
  * @param key_size   length of the encryption key
+ *
+ * @return  1 if the initialization was successful or 0 in case of an error. 
```

Comment:
Added define for successful initialization. While I was at it, I also added a new error code for the 0 rv returned when the context size is too small.